src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerPostAdminApproveMediaMarkedForDeletionIntTest.java

```diff
@@ -127,6 +128,16 @@ void postAdminApproveMediaMarkedForDeletionShouldReturnSuccess() throws Exceptio
                 criteriaBuilder.equal(root.get(AuditEntity_.auditActivity).get("id"), AuditActivity.MANUAL_DELETION.getId())
             ));
 
+        // assert additional audit data
+        assertFalse(caseExpiredAuditEntries.isEmpty());
+        assertNotNull(caseExpiredAuditEntries.get(0).getCreatedBy());
+        assertNotNull(caseExpiredAuditEntries.get(0).getLastModifiedBy());
+        assertNotNull(caseExpiredAuditEntries.get(0).getCreatedDateTime());
+        assertNotNull(caseExpiredAuditEntries.get(0).getLastModifiedDateTime());
+        assertNotNull(caseExpiredAuditEntries.get(0).getLastModifiedDateTime());
+        assertEquals(userIdentity.getUserAccount().getId(), caseExpiredAuditEntries.get(0).getUser().getId());
+        assertNull(caseExpiredAuditEntries.get(0).getCourtCase());
+
         assertFalse(caseExpiredAuditEntries.isEmpty());
```

Comment:
Not needed as verified on line 132

Reply:
Could you check this again. The line numbers do not seem relevant

Reply:
Agreed I no longer see the relevant to this comment (Guessing code change to mark it redundant) happy for this to be closed

Reply:
Line 132 and line 141 seem to be doing the same thing only one of these verification points is needed

Reply:
Done